### ISSUE TITLE
Lack of properties for Accessibility

### ISSUE THREAD

**Issue by lrosenthol** [google/A2UI#307]
Many of the standard components will need to have accessibility information provide with them in order to expose UX's that can be used by everyone.  This spans the gamut from alt-text on images to aria-roles on buttons to captioning for videos.

Would be good to start with something as simple as an "alt-text" property in the common set for all components.

**Comment by jacobsimionato** [comment]
This is super important - thanks for filing! We have added essentially what you said in https://github.com/google/A2UI/pull/527, so I'll close this.

Please do reopen or file more issues if you find additional a11y issues.
